src/tools/wifi-qr-code-generator/wifi-qr-code-generator.vue

```diff
@@ -0,0 +1,153 @@
+<script setup lang="ts">
+import {
+  EAPMethods,
+  EAPPhase2Methods,
+  useWifiQRCode,
+} from './useQRCode';
+import { useDownloadFileFromBase64 } from '@/composable/downloadBase64';
+
+const foreground = ref('#000000ff');
+const background = ref('#ffffffff');
+
+const ssid = ref();
+const password = ref();
+const eapMethod = ref();
+const isHiddenSSID = ref(false);
+const eapAnonymous = ref(false);
+const eapIdentity = ref();
+const eapPhase2Method = ref();
+
+const { qrcode, encryption } = useWifiQRCode({
+  ssid,
+  password,
+  eapMethod,
+  isHiddenSSID,
+  eapAnonymous,
+  eapIdentity,
+  eapPhase2Method,
+  color: {
+    background,
+    foreground,
+  },
+  options: { width: 1024 },
+});
+
+const { download } = useDownloadFileFromBase64({ source: qrcode, filename: 'qr-code.png' });
+</script>
+
+<template>
+  <c-card>
+    <n-grid x-gap="12" y-gap="12" cols="1 600:3">
```

Comment:
I'm triyng to get rid of naive-ui (see #400 ) can you use unocss `grid` class instead of n-grid ?  🙏🏻 
If needed, I can do it myself

Reply:
I can work on that tonight